content/docs/authorization/cli.md

```diff
@@ -260,14 +260,14 @@ karavictl role get [flags]
 
 ```
   -h, --help   help for get
-      --insecure           insecure skip verify flag for Helm deployment
-      --addr               address of the container for Helm deployment (pod:port)
 ```
 
 ##### Options inherited from parent commands
 
 ```
       --config string   config file (default is $HOME/.karavictl.yaml)
+      --insecure           insecure skip verify flag for Helm deployment
+      --addr               address of the container for Helm deployment (pod:port)
```

Comment:
Let's keep these flag descriptions consistent with tenant and storage commands.

```
--addr string     Address of the server (default "localhost")
--insecure        Skip certificate validation
```

Reply:
fixed